DO NOT MERGE: Tmp azure all tests

smg260 opened the pull request:
TEST ONLY

Commit:
TMP run all gce tests on azure too and exlicitly disable issue posting (should not post anyway)

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/111782)
<!-- Reviewable:end -->

Commit:
roachprod: refactor azure security rules and open ports

This adds some more ports required for various roachtests and
refactors their creation with a helper function.

Epic: CC-25185

Release note: None

Commit:
upload stats on this branch only

Commit:
roachprod: add azure 96cpu machine type

Epic: CC-25185

Release note: None

Commit:
roachtest: apt-get update for asyncpg

This is required so that packages can be found in azure.

Epic: CC-25185

Release note: None

Commit:
roachtest: add support for azure premium/ultra disks

Epic: CC-25185
Release note: none

Commit:
roachprod: set azure default location westus2

`westus` does not support availability zones, but
`westus2` does. Over 20 roachtests use az.

Epic: CC-25185
Release note: none

cockroach-teamcity commented:
[![CLA assistant check](https://cla.crdb.dev/pull/badge/not_signed)](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=111782) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=111782) before we can accept your contribution.<br/><hr/>**Miral Gadani** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.crdb.dev/check/cockroachdb/cockroach?pullRequest=111782) it.</sub>
